pom.xml

```diff
@@ -1619,6 +1621,7 @@ under the License.
 											so there's no benefit in us investing time into bumping these. -->
 										<include>org.yaml:snakeyaml:(,1.26]:*:test</include>
 									</includes>
+									<message>Older snakeyaml versions are not allow due to security vulnerabilities.</message>
```

Comment:
Could maybe be worthwhile specifying the minimum version in the message? Like "snakeyaml versions older than x.y", just for friendliness.

Reply:
It would be neat but I would like to avoid duplicating the version information, which we then of course would have to keep in sync.
we could of course define some `security.snakeyaml.minimum.version` property which we use, but it seems overkill?

Reply:
Good point! Yeah, I think that would be overkill.